app/helpers/ebird_helper.rb

```diff
@@ -83,6 +83,7 @@ def getImageSrc(bird_data)
 
   def getImageFromName(name)
 
+    #check cache for bird name
```

Comment:
Is this meant to be a TODO? If so, write # TODO: Check cache for bird name
It looks more like pseudocode that I would write when planning a feature and then replace later while coding, is it meant to be checked in? (I wouldnt typically)